openwisp_monitoring/notifications/templates/notifications/threshold_crossed.md

```diff
@@ -0,0 +1,7 @@
+{% extends 'openwisp_notifications/default_message.md' %}
```

Comment:
we should move this stuff to `openwisp_monitoring.monitoring`. I realized `openwisp_monitoring.notifications` should be deleted to avoid confusion. We have openwisp_monitoring.notifications importing openwisp_notifications which in turn imports django-notification-hq, lol.